### PR TITLE
Remove audio logo from NVDA launcher

### LINKED ISSUE #14068: Use shorter startup sound for NVDA launcher
### Is your feature request related to a problem? Please describe.

The startup music played by the NVDA launcher (the file you download from https://www.nvaccess.org/download/ ) is relatively long.  On fast computers, this may take longer to play than the time taken to load the temporary copy of NVDA.

### Describe the solution you'd like

A shorter intro sound, could either play once, or loop for as long as it takes to load NVDA.

### Describe alternatives you've considered

We could play the progress bar sound, or a short intro sound followed by the progress bar sound, although this may be confusing for new users, as the same progress bar sound is used later when installing NVDA..

### LINKED ISSUE #15110: Add ability to silence NVDA installer startup sound when updating NVDA (request from user)
### Is your feature request related to a problem? Please describe.

Reporting user (on Mastodon - https://fosstodon.org/@Brynify@dragonscave.space/110685946192113063 ) indicates they don't like having the installer startup sound while updating and would like an option to silence it like the startup sound option.

### Describe the solution you'd like

Could possibly bundle this WITH the startup sound option?

### PR DESCRIPTION
**Pull request by SaschaCowley**
### Link to issue number:

Closes #14068
Closes #15110

### Summary of the issue:

The NVDA launcher currently plays an audio logo when launched.
This creates a number of problems:

* Playback of this file uses winmm, which is a deprecated API.
* Playback of this file may delay execution of the launcher, if it takes longer than unpacking the launcher does. (#14068)
  * The NVDA audio logo is 10 seconds long, whereas it typically takes around 6 seconds for the launcher to start the temporary NVDA on my machine.
* Some users may want to silence playback of this sound altogether, which is currently not possible. (#15110)

### Description of user facing changes

The NVDA launcher is now silent until the temporary copy of NVDA it includes is started.

### Description of development approach

Removed all code related to playing the audio logo from `launcher/launcher.nsis`.

### Testing strategy:

Built the launcher and ran it from explorer.
Also ran NVDA from source patched to check for updates, and swapped the downloaded launcher for the one without the sound logo.

### Known issues with pull request:

There is now an appreciable period of silence between NVDA quitting to perform an update, and the temporary NVDA speaking to read out the update progress.

### Code Review Checklist:

- [x] Documentation:
  - Change log entry
  - User Documentation
  - Developer / Technical Documentation
  - Context sensitive help for GUI changes
- [x] Testing:
  - Unit tests
  - System (end to end) tests
  - Manual testing
- [x] UX of all users considered:
  - Speech
  - Braille
  - Low Vision
  - Different web browsers
  - Localization in other languages / culture than English
- [x] API is compatible with existing add-ons.
- [x] Security precautions taken.

@coderabbitai summary

### CODE REVIEW

**Comment by cary-rowen** [comment]
> There is now an appreciable period of silence between NVDA quitting to perform an update, and the temporary NVDA speaking to read out the update progress.

Can we improve this? Is it possible to use a shorter sound? In order to let users know that the update has started, rather than being stuck for some reason.

File: launcher/nvdaLauncher.nsi [hunk 1]
```diff
@@ -11,7 +11,6 @@ CRCCheck on
 
 ReserveFile "${NSISDIR}\Plugins\x86-unicode\system.dll"
 ReserveFile "${NSISDIR}\Plugins\x86-unicode\banner.dll"
-ReserveFile "..\miscDeps\launcher\nvda_logo.wav"
```
Comment (seanbudd) [inline launcher/nvdaLauncher.nsi:14]: can we also delete that file from `miscDeps` in a separate PR?

**Review by seanbudd — APPROVED** [review]

File: launcher/nvdaLauncher.nsi (hunk 1 above)
Comment (SaschaCowley) [reply launcher/nvdaLauncher.nsi:14]: Created nvaccess/nvda#35. Do we want to merge that and update the submodule on this branch?

**Comment by SaschaCowley** [comment]
@cary-rowen
> > There is now an appreciable period of silence between NVDA quitting to perform an update, and the temporary NVDA speaking to read out the update progress.
> 
> Can we improve this? Is it possible to use a shorter sound? In order to let users know that the update has started, rather than being stuck for some reason.

Unfortunately `PlaySound` and friends are the only easy ways of natively playing a sound file that I'm aware of, and they all use winmm. As we're running under NSIS, our options here are rather limited.

**Comment by dpy013** [comment]
Can we take a look at Inno Setup?

**Comment by seanbudd** [comment]
@dpy013 - we do not intend to add back a launcher sound or migrate from NSIS for the time being. I'd encourage dropping the concern

**Comment by retrochiptuner** [comment]
Hello! I agree that this is not a good alternative. In any case with a shorter start sound. Anyway on my computer the installer of the last 2026.1 Beta 1 and the last alphas also take time to load the temporary copy. My computer is a ASUS X541UAK with an Intel (R) Core (TM) I3-6006U cpu @ 2.00GHZ, 1992 MHz, 2 main processors, 4 logical processors.

**Pull request merged**